ga4gh/frontend.py

```diff
@@ -111,11 +149,16 @@ def handleException(exception):
     return response
 
 
-def handleFlaskRequest(flaskRequest, endpoint):
+def handleFlaskPostRequest(version, flaskRequest, endpoint):
+    if Version.parseString(version) != Version.parseString(protocol.version):
```

Comment:
Why not `version != protocol.version`? It's easy to make this more general when we need it.